language/move-prover/tests/sources/functional/fixed_point_arithm.move

```diff
@@ -123,10 +125,26 @@ module 0x42::FixedPointArithmetic {
         FixedPoint32::multiply_u64(z, FixedPoint32::create_from_raw_value(y_raw_val))
     }
     spec div_mul_incorrect {
+        pragma verify=false; // TODO: disabled due to the CVC4 timeout
```

Comment:
~An alternative would be to have `// separate_baseline: cvc4` in the beginning of the file followed by a `TODO` mark. So when cvc4 support is ready, we can remove that line.~

Since this is already there, maybe we should remove the `verify = false`?

Reply:
I think it may be better to keep the `verify=false` in this case. It's because such timeouts can make local `cargo test` slow for everyone. Moreover, in this specific case, Prover does not terminate with a timeout error message, but just hang forever. I will file a github issue for this.